python/pyspark/ml/tests/test_wrapper.py

```diff
@@ -116,9 +116,8 @@ def test_new_java_array(self):
         java_array = JavaWrapper._new_java_array(str_list, java_class)
         self.assertEqual(_java2py(self.sc, java_array), expected_str_list)
 
-if __name__ == "__main__":
-    from pyspark.ml.tests.test_wrapper import *
```

Comment:
So .. let's don't remove this.

Reply:
Just a small spike, I believe this is redundant. You're importing everything from the file itself. Let's wait for Jenkins to see if the file is being reported by Jenkins unit test report. If so, the import is not needed, if not, then it is required.